File: Practice(Beginner)/Python3/Chef_and_SnackDown.py
# Source: https://www.codechef.com/problems/SNCKYEAR

t = int(input())
for test in range(t):
    n = int(input())
    print("HOSTED") if (n == 2010 or n == 2015 or n == 2016 or n == 2017 or n == 2019) else print("NOT HOSTED")

File: Practice(Beginner)/C++/Chef and SnackDown.cpp
// Source: https://www.codechef.com/problems/SNCKYEAR

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;
    while(t--){
        int n;
        cin >> n;
        n == 2010 or n == 20015 or n == 2016 or n == 2017 or n == 2019 ? cout << "HOSTED" << endl : cout << "NOT HOSTED" << endl;
    }
}